### ISSUE TITLE
Unit test 'rtcrtpreceiver.js' crashes

### ISSUE THREAD

**Issue by sebas642** [node-webrtc/node-webrtc#432]
The `rtcrtpreceiver.js` test crashes every single time I run it. The backtrace is the following:
```text
#0  0x00007f7895563f23 in std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::string const&) ()
   from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#1  0x00007f7891bc9782 in webrtc::RtpCodecParameters::RtpCodecParameters (this=0x7f78341aa388)
    at ../third_party/webrtc/include/webrtc/api/rtpparameters.h:340
#2  0x00007f7891d4feeb in void std::vector<webrtc::RtpCodecParameters, std::allocator<webrtc::RtpCodecParameters> >::_M_emplace_back_aux<webrtc::RtpCodecParameters>(webrtc::RtpCodecParameters&&) () from /tmp/node-webrtc/build/Debug/wrtc.node
#3  0x00007f7891d476b0 in cricket::WebRtcVoiceMediaChannel::GetRtpReceiveParameters(unsigned int) const ()
   from /tmp/node-webrtc/build/Debug/wrtc.node
#4  0x00007f7891e2aac2 in cricket::VoiceChannel::GetRtpReceiveParameters_w(unsigned int) const () from /tmp/node-webrtc/build/Debug/wrtc.node
#5  0x00007f7891e3524b in rtc::FunctorMessageHandler<webrtc::RtpParameters, rtc::MethodFunctor<cricket::VoiceChannel const, webrtc::RtpParameters (cricket::VoiceChannel::*)(unsigned int) const, webrtc::RtpParameters, unsigned int> >::OnMessage(rtc::Message*) ()
   from /tmp/node-webrtc/build/Debug/wrtc.node
#6  0x00007f7891c97370 in rtc::Thread::ReceiveSendsFromThread(rtc::Thread const*) () from /tmp/node-webrtc/build/Debug/wrtc.node
#7  0x00007f7891dd2807 in rtc::MessageQueue::Get(rtc::Message*, int, bool) () from /tmp/node-webrtc/build/Debug/wrtc.node
#8  0x00007f7891c96f12 in rtc::Thread::Run() () from /tmp/node-webrtc/build/Debug/wrtc.node
#9  0x00007f7891c96c24 in rtc::Thread::PreRun(void*) () from /tmp/node-webrtc/build/Debug/wrtc.node
#10 0x00007f7894cbf6db in start_thread (arg=0x7f77f53ef700) at pthread_create.c:463
#11 0x00007f78949e888f in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```
`valgrind` gives the same backtrace with the following error:
```text
==7090== Invalid read of size 4
==7090==    at 0x531EF23: std::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::string const&) (in /usr/lib/x86_6
4-linux-gnu/libstdc++.so.6.0.25)
==7090==    by 0x9AB5781: webrtc::RtpCodecParameters::RtpCodecParameters(webrtc::RtpCodecParameters const&) (rtpparameters.h:340)
...
==7090==  Address 0xfffffffffffffff8 is not stack'd, malloc'd or (recently) free'd
```

This is with the following environment:
- Node v8.11.2
- Ubuntu Linux 18.04
- wrtc v0.2.0

**Comment by sebas642** [comment]
I can reproduce this with v0.1.4 as well.

**Comment by markandrus** [comment]
I think this will be fixed in v0.3.0 (#445).
